web/packages/teleport/src/components/ReAuthenticate/ReAuthenticate.tsx

```diff
@@ -69,68 +70,71 @@ export function ReAuthenticate({
       {({ validator }) => (
         <Dialog
           dialogCss={() => ({
-            width: '400px',
+            width: '416px',
           })}
           disableEscapeKeyDown={false}
           onClose={onClose}
           open={true}
         >
-          <DialogHeader style={{ flexDirection: 'column' }}>
-            <DialogTitle>Verify your identity</DialogTitle>
-            <Text textAlign="center" color="text.secondary">
-              You must verify your identity before peforming this action.
-            </Text>
-          </DialogHeader>
-          {attempt.status === 'failed' && (
-            <Danger mt={2} width="100%">
-              {attempt.statusText}
-            </Danger>
-          )}
-          <DialogContent>
-            <Flex mt={2} alignItems="flex-end">
-              <FieldSelect
-                width="50%"
-                label="Two-factor type"
-                value={mfaOption}
-                options={mfaOptions}
-                onChange={(o: MfaOption) => {
-                  setMfaOption(o);
-                  clearAttempt();
-                }}
-                data-testid="mfa-select"
+          <form>
+            <DialogHeader style={{ flexDirection: 'column' }}>
+              <DialogTitle>Verify your identity</DialogTitle>
+              <Text textAlign="center" color="text.secondary">
+                {dialogText}
```

Comment:
Super nit: I'd consider hardcoding the part that doesn't change ("You must verify your identity with one of your existing two-factor devices before") and letting the callsites change only the last part.